lambda/es-proxy-layer/lib/es_query.js

```diff
@@ -0,0 +1,56 @@
+
+let request = require('./request');
+let build_es_query = require('./esbodybuilder');
+let _ = require('lodash')
+
+
+async function run_query_es(req, query_params) {
+    
+    var es_query = await build_es_query(query_params);
+    var es_response = await request({
+        url: `https://${req._info.es.address}/${req._info.es.index}/_doc/_search?search_type=dfs_query_then_fetch`,
```

Comment:
Consider doing additional input validation. Specifically on the parameters / url. This would be additional guard again miss-configuration or accessing unintended sites. For example, we can check `req._info.es.address` is a valid opensearch endpoint.

Multiple occurrence.

If worthwhile, this could be in future backlog enhancement.